src/app/site/about-us/about-us.component.ts

```diff
@@ -20,7 +21,8 @@ export class AboutUsComponent implements OnInit {
     this.waiting = true;
     this.getJsonFileService.getAboutUsData()
       .then((details) => {
-        this.aboutUs = details[0];
+        this.result = details;
```

Comment:
do we really needed  result?
we can use details instead of result

Reply:
yes it's needed, so I've used result[0] , result[1] and result[2] in .html file as input for child components, while .html file does not know details(its the argument of then, and is not reachable in html file)